<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransForm</class>
 <widget class="QWidget" name="TransForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>270</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="transHorizontalWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>250</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="verticalWidget_3" native="true">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignLeft|Qt::AlignTop">
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Transform Component</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="translateLayout">
          <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="translateLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Translate</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="translateWidget" native="true">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="xTransLabel">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="xTransSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="yTransLabel">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="yTransSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="zTransLabel">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="zTransSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rotationLayout">
          <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="rotationLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Rotation</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="rotationWidget" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="xRotLabel">
               <property name="text">
                <string> X:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="xRotSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="yRotLabel">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="yRotSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="zRotLabel">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="zRotSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ScaleLayout">
          <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="scaleWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="ScaleWidget" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="xScaleLabel">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="xScaleSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="yScaleLabel">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="yScaleSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="zScaleLabel">
               <property name="text">
                <string>Z:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QDoubleSpinBox" name="zScaleSpinBox">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
